pandas/tests/test_resample.py

```diff
@@ -2719,6 +2719,11 @@ def test_resample_weekly_bug_1726(self):
         # it works!
         df.resample('W-MON', closed='left', label='left').first()
 
+    def test_resample_tz_aware_bug_15549(self):
+        index = pd.DatetimeIndex([1450137600000000000, 1474059600000000000], tz='UTC').tz_convert('America/Chicago')
```

Comment:
Reference issue number.

Reply:
sorry, but I already referenced it in the function name or you mean something else?